owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -1300,6 +1295,10 @@ class FileDisplayActivity : FileActivity(),
                 waitingToSend = null
                 waitingToOpen = null
             },
+            onWorkCancelled = {
+                waitingToSend = null
+                waitingToOpen = null
+            },
```

Comment:
Chekcing that a little bit deeper this, Im not completely sure. If we reset completely the waiting to send or open, if could affect even the ones that are not being cancelled, so Im not sure 100% about this

Reply:
I think I fixed it 😀